doc/source/whatsnew/v1.2.0.rst

```diff
@@ -267,6 +267,7 @@ Sparse
 ExtensionArray
 ^^^^^^^^^^^^^^
 
+- Added :meth:`api.extensionExtensionArray._accumulate` to the extension array interface. Implements this interface for :class: `IntegerArray` and :class: `BooleanArray` such that type coercion to `object` is avoided (:issue:`28385`)
```

Comment:
can you instead frame this as what a user would understand as `_accumulate` is a private method